### ISSUE TITLE
Z-Wave Unresponsive/Stuck - Danfoss LC-13 E5?

### ISSUE THREAD

**Issue by mike9011** [athombv/homey#1125]
**What is your Homey version (Settings → System)?**

1.0.2

**What did you try to achieve?**
Make use of Z-Wave devices.

**Please include detailed steps how we can reproduce the issue**
Got the E5 error again on the Danfoss LC-13, after this error i'm not able to control any other z-wave devices e.g. Double Switch 2. 

It remains continuously on:
currentCommand: SendData
currentProcess: ProcessSendData

![screenshot_20161201-215957](https://cloud.githubusercontent.com/assets/18025163/20825990/27651bae-b868-11e6-8a9e-4220f87bb4e9.png)

`{"zw_state":{"currentProcess":"ProcessSendData","ver":5,"capabilities":{"api":"controller","timerSupported":false,"controllerType":"primary"},"nodes":[1,10,13,14,15,16,19,20,24],"chip_type":5,"chip_version":0,"homeId":3481936477,"nodeId":1,"nvr":[1,2,0,21,13,27,120,32,1,0,32,0,64,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,23,19,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,68,155],"CONTROLLER_IS_SECONDARY":false,"CONTROLLER_ON_OTHER_NETWORK":false,"CONTROLLER_NODEID_SERVER_PRESENT":true,"CONTROLLER_IS_REAL_PRIMARY":true,"CONTROLLER_IS_SUC":true,"NO_NODES_INCUDED":false,"sucId":1,"powerLevel":0,"version":"Z-Wave 3.99","networkKey":"4c3852a3312fbd22cb6be8d9f897457a","methodsAsync":["factoryReset","addNodeAbort","removeNodeAbort","replaceFailedNodeAbort","sendData","isFailedNode","getFailedNodes"],"currentCommand":"sendData"},"zw_ready":true}`

**What was the result?**
Z-wave Stuck - PTP needed to restore the functionality.

**What did you expect as result?**

**Any other remarks**

...

**Comment by MarcelAkerboom** [comment]
+1 on this. 

Z-Wave regularly gets stuck on SendData, only PTP temporarily fixes the issue. 
One difference though, I only have Fibaro and Aeotec devices connected.

After seeing a possible fix in the changelog of 1.0.3 experimental, I enabled the exp-mode. 
This seemed to fix it, only after 6 hours later Homey got stuck again.

**Comment by bvdbos** [comment]
E5 is fixed for Devolo, maybe the Danfoss-app still needs some love?

**Comment by MarcelAkerboom** [comment]
Personally, I think this issue is unrelated to any specific app. 
Since I do not use Devolo or Danfoss but have the same symptoms.

Yesterday I had a kind of a hint towards the origin of the symptoms. 
I had been adding/removing z-wave devices (Fibaro) through the chrome browser. But removing kept getting stuck. So I opened up the Homey desktop application and went to the z-wave settings page.

There was a long queue of remove-node commands, had to click cancel about 10 times and like magic, all my previous light commands went through (on-off-on-off-dim-on-off) haha.
So, somehow the z-wave controller gets stuck waiting on a command that does never finish (for me removing device), and so all other commands seem to get queued up.

Anyway. after not using the browser to do these actions, this issue has not occurred anymore.

**Comment by PandaNL** [comment]
Same issue here, after 1.0.1 danfoss works great, but since 1.0.2 getting the E5 error again.

Another issue is that the voice command set temperature doesn't seem to work for me, Homey receives the voice input correctly but won't act to it.

**Comment by jhutjens** [comment]
+1 on this one. Same situation. E5 error on all three lc-13 thermostats directly after pairing or after a short while. After that can't find signal anymore and need to be repaired but does not fix the problem. Also using a z-wave repeater from aeotec

**Comment by RobinBol** [comment]
E5 issue will be continued here: https://github.com/athombv/homey/issues/1187
